quartz/plugins/transformers/ofm.ts

```diff
@@ -346,11 +352,32 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
               }
             })
           }
-
           mdastFindReplace(tree, replacements)
         }
       })
 
+      //go through mdast, find img nodes with video extension, convert them to html nodes
+      if (opts.enableVideoEmbed) {
+        plugins.push(() => {
+          return (tree: Root, _file) => {
+            visit(tree, "image", (node, index, parent) => {
+              const match = node.url.match(videoExtensionRegex)
+              if (match && parent) {
+                const htmlNode: PhrasingContent = {
+                  type: "html",
+                  value: `<video controls autoplay src="${node.url}" controls></video>`,
```

Comment:
```suggestion
                  value: `<video controls src="${node.url}" controls></video>`,
```

let's not autoplay by default